src/Microsoft.Net.Http.Headers/EntityTagHeaderValue.cs

```diff
@@ -103,7 +103,7 @@ public override bool Equals(object obj)
             }
 
             // Since the tag is a quoted-string we treat it case-sensitive.
-            return ((_isWeak == other._isWeak) && (string.CompareOrdinal(_tag, other._tag) == 0));
```

Comment:
Any reason we were going in a roundabout way of comparing strings previously? Why `CompareOrdinal` instead of `Equals`?